### PR TITLE
Converted ZC_EFST_SET_ENTER to struct

### PR DESCRIPTION
**Pull request by AoShinRO**
* **Addressed Issue(s)**: 

* **Server Mode**:  both

* **Description of Pull Request**:  PACKET_ZC_EFST_SET_ENTER

### CODE REVIEW

File: src/map/packets.hpp [hunk 1]
```diff
@@ -914,6 +914,31 @@ struct PACKET_ZC_NOTIFY_EFFECT {
...
+DEFINE_PACKET_HEADER(ZC_EFST_SET_ENTER, 0x984)
+#elif PACKETVER >= 20111108
+struct PACKET_ZC_EFST_SET_ENTER{
+	uint16 packetType;
+	uint32 tid;
+	uint16 type;
+	uint32 duration;
+	uint32 val1;
+	uint32 val2;
+	uint32 val3;
```
Comment (Atemo) [inline src/map/packets.hpp:937]: For me valx should not be unsigned even though I don't remember a case where it is negative in our sc

Comment (AoShinRO) [reply src/map/packets.hpp]: i'm in change of this now because i asked same to myself 😄 but I was going to ask for confirmation here

**Review by aleos89 — APPROVED** [review]

File: src/map/packets.hpp [hunk 2]
```diff
@@ -941,6 +941,31 @@ struct PACKET_ZC_NOTIFY_EFFECT {
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_NOTIFY_EFFECT, 0x19b);
 
+#if PACKETVER >= 20120618
+struct PACKET_ZC_EFST_SET_ENTER{
+	uint16 packetType;
+	int srcId;
```
Comment (AoShinRO) [inline src/map/packets.hpp:947]:
```suggestion
	uint32 srcId;
```

File: src/map/packets.hpp [hunk 3]
```diff
@@ -941,6 +941,31 @@ struct PACKET_ZC_NOTIFY_EFFECT {
...
+	uint32 duration2;
+	int val1;
+	int val2;
+	int val3;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_EFST_SET_ENTER, 0x984)
+#elif PACKETVER >= 20111108
+struct PACKET_ZC_EFST_SET_ENTER{
+	uint16 packetType;
+	int srcId;
```
Comment (AoShinRO) [inline src/map/packets.hpp:959]:
```suggestion
	uint32 srcId;
```

**Pull request closed without merging**